app/js/services.js

```diff
@@ -1,9 +1,10 @@
-var conceptDirServices = angular.module('conceptDirServices', ['ngResource']);
+var conceptDictServices = angular.module('conceptDictServices', ['ngResource']);
 
-conceptDirServices.factory('Classes', ['$resource',
+
+conceptDictServices.factory('Classes', ['$resource',
                                    function($resource){
                                      return $resource(
-                                    		 'http://localhost:8080/openmrs/ws/rest/v1/conceptclass/8d4907b2-c2cc-11de-8d13-0010c6dffd0f', {}, {
+                                    		 'http://'+ location.host +'/openmrs/ws/rest/v1/conceptclass?v=full', {}, {
```

Comment:
We shouldn't be declaring 'http://'+ location.host +'. One reason is that it may be https. The other reason is it is not needed. It should be enough to put '/openmrs/ws/rest/v1/conceptclass?v=full' for this to work. However, as I wrote previously '/openmrs' can change too so it is still not ideal. It is best we store '/openmrs' in a global variable. We can write some js code to determine that part of the URL, but we can leave that for later and start with the global variable.